js/repositories.bzl

```diff
@@ -25,3 +26,7 @@ def rules_js_dependencies():
         strip_prefix = "bazel-lib-1.32.1",
         url = "https://github.com/aspect-build/bazel-lib/releases/download/v1.32.1/bazel-lib-v1.32.1.tar.gz",
     )
+
+    _config_settings(
+        name = "aspect_config_settings",
```

Comment:
this is creating a repo for rules_js in a global namespace of workspaces, so I think the name needs to be fixed like aspect_rules_js_config_settings? What about aspect_bazel_lib as a home for this? I assume we'll have a problem with this flag rename in some of our other rulesets

Reply:
Yeah I don't think this location is where this should live long term. I'm not sure if we want to release a bazel-lib change or get this fixed within rules_js asap?

Reply:
Or it might actually have to be here due to what @thesayyn was pointing out.

I've done the name change though.